sympy/stats/crv_types.py

```diff
@@ -2459,6 +2531,71 @@ def LogNormal(name, mean, std):
 
     return rv(name, LogNormalDistribution, (mean, std))
 
+#-------------------------------------------------------------------------------
+# Lomax Distribution -----------------------------------------------------------
+
+class LomaxDistribution(SingleContinuousDistribution):
+    _argnames = ('alpha', 'lamda',)
+    set = Interval(0, oo)
+
+    @staticmethod
+    def check(alpha, lamda):
+        _value_check(alpha.is_real, "Shape parameter should be real.")
+        _value_check(lamda.is_real, "Scale parameter should be real.")
+        _value_check(alpha.is_positive, "Shape parameter should be positive.")
+        _value_check(lamda.is_positive, "Scale parameter should be positive.")
+
+    def pdf(self, x):
+        lamba, alpha = self.lamda, self.alpha
+        return (alpha/lamba) * (S.One + x/lamba)**(-alpha-1)
```

Comment:
A minor typo but can be ignored.

Reply:
@Smit-create Can you correct this typo because I don't think we should compromise with code quality here.

Reply:
typo of using `1` intead of `S.One`?

Reply:
typo of `lamba`->`lamda`

Reply:
Oops, will change that in my next PR for sure. Thanks!!

Reply:
Cool!!